internal/fptest/networkinstance.go

```diff
@@ -43,3 +45,29 @@ func AssignToNetworkInstance(t testing.TB, d *ondatra.DUTDevice, i string, ni st
 		gnmi.Update(t, d, gnmi.OC().NetworkInstance(ni).Config(), netInst)
 	}
 }
+
+// EnableGribiOnNetworkInstance enables Gribi protocol under network instance.
+func EnableGribiUnderNetworkInstance(t testing.TB, d *ondatra.DUTDevice, ni string) {
+	t.Helper()
+	if ni == "" {
+		t.Fatalf("Network instance not provided for protocol definition")
+	}
+	cliCmd := fmt.Sprintf("/set network-instance %s protocols gribi admin-state enable", ni)
```

Comment:
Marc -- do you prefer to do this in the SRLinux model vs. CLI? Per our discussion given that this is behind a deviation this seems OK (and we should figure out the plan here), but I'm not sure this _must_ be CLI :-)

Reply:
@robshakir, if we're allowed to use the native model for this deviation, I'd prefer that. I'll work on a commit to migrate it. cc @xw-g